src/app/app.component.ts

```diff
@@ -7,4 +8,14 @@ import { Component } from '@angular/core';
 })
 export class AppComponent {
   title = 'HomeworkAngularDs';
+  value='true';
+  active = true; 
+  changeColor() { 
+      this.active = !this.active; 
+      if (!this.active) { 
```

Comment:
esto se puede cambiar por un oneliner así: this.value = this.active ? 'true' : 'false';